src/common/access_pattern.c

```diff
@@ -350,11 +352,15 @@ xdd_save_seek_list(target_data_t *tdp) {
 		}
 		average = (uint64_t)(total / sp->seek_total_ops);
 		/* Print the seek list into the specified file */
+		fprintf(tmp, "# Block size=%d, Request size=%d, Numreqs=%ld\n",
+                (int32_t)tdp->td_block_size,
+                (int32_t)tdp->td_reqsize,
+                (long int)tdp->td_numreqs);
```

Comment:
We only should be putting the number of request in the seek header. The block size and request size can be determined by the seek file contents themselves as those values are already there for any any I/O operation anyways. This will also allow for future work were we have xdd do varying block sizes for a target. The that info can be gleaned from the rest of the seek file.